sources/core/objects/Human.cpp

```diff
@@ -38,7 +39,10 @@ Human::Human(quint32 id) : IMob(id)
 
     dead_ = false;
     lying_ = false;
-    health_ = 100;
+    max_health_ = 10000;
```

Comment:
Almost all literals (numbers/strings) should be used only as constant.
Here should be
```
max_health_ = HUMAN_MAX_HEALTH;
```
`HUMAN_MAX_HEALTH` may be defined in `Human.h`.